new/PRS14101Baselyos.xml

```diff
@@ -47,11 +47,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person sex="1">
                     <persName xml:lang="gez" xml:id="n1">ባስልዮስ</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1">Bāsəlyos</persName>
-                    <persName><roleName type="title">ʾabbā</roleName></persName>
+                    <persName xml:lang="gez" type="normalized"><roleName type="title">ʾabbā</roleName></persName>
```

Comment:
here it was actually correct - if you do not have the `lang `then you do not need the `type`

Reply:
OK. I will remember in the future.